### PR TITLE
[MIG] purchase_stock_analytic: Migration to 16.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
@qrtl

### CODE REVIEW

File: purchase_stock_analytic/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+To use this module, you need to:
+
+#. Create a purchase order
+#. Set analytic distribution on purchase order lines
+#. Confirm purchase order
```
Comment (yostashiro) [inline purchase_stock_analytic/readme/USAGE.rst:5]:
```suggestion
#. Confirm purchase order
#. ... and you should have your analytic account and analytic tags on your stock moves
```
According to the readme of 14.0 module.

**Comment by yostashiro** [comment]
Loosely related (for the validation): https://github.com/OCA/account-analytic/pull/543

**Review by yostashiro — APPROVED** [review]
Functional review.

**Review by ghost — APPROVED** [review]

**Pull request merged**